s64da_benchmark_toolkit/db.py

```diff
@@ -99,3 +99,7 @@ def get_explain_output(connection, sql):
 
         except psycopg2.Error as e:
             return f'{{"Explain Output failed": "{str(e)}"}}'
+
+        except json.JSONDecodeError as e:
+            LOG.error('Explain Output failed with a JSON Decode Error')
+            return f'{{"Explain Output failed with a JSON Decode Error": "{str(e)}"}}'
```

Comment:
Are these `{{` intentional?

Reply:
Please check, whether `LOG.error` stops everything and maybe use `LOG.warning` instead.

Reply:
I checked it runs ok, but the '{{' can be removed was just following the syntax used for the above psycopg2 error

Reply:
Yes, I saw that. But it is not needed except you want to print `{` as character.

Reply:
I think it makes sense to use `LOG.warning` and a slightly different message, as it is not a fatal error or the like. But this is just my opinion.